src/relay.cpp

```diff
@@ -827,67 +829,29 @@ void server_callback(evutil_socket_t fd, short event, void *arg) {
     socklen_t len = sizeof(from);
     int32_t data = 0;
     static uint8_t message_buffer[BUFFER_SIZE];
+    std::string counterVlan = counter_table;
 
     if ((data = recvfrom(config->local_sock, message_buffer, BUFFER_SIZE, 0, (sockaddr *)&from, &len)) == -1) {
         syslog(LOG_ERR, "recv: Failed to receive data from server\n");
         return;
     }
 
     if (data < (int32_t)sizeof(struct dhcpv6_msg)) {
-        syslog(LOG_WARNING, "Invalid DHCPv6 header");
-        return;
+        syslog(LOG_WARNING, "Invalid DHCPv6 packet length %d, no space for dhcpv6 msg header\n", len);
```

Comment:
here should use data, len is socket address length. any same as above %zu maybe better for size_t

Reply:
Thanks for catching that, updated